docs/src/pages/_app.page.tsx

```diff
@@ -1,5 +1,7 @@
 import * as React from 'react';
 
+import { Amplify } from 'aws-amplify';
+
```

Comment:
nit: we do not need a line break here, since both `aws-amplify` and `@aws-amplify/ui-react` are internal dependencies. They should be grouped together

Reply:
Done!